devel/zmqpp/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        zeromq zmqpp 4.2.0
+categories          devel net
+platforms           darwin
+license             MPL-2.0
+maintainers         {@ierofant gmail.com:aw.kornilov} openmaintainer
+
+description         0mq 'highlevel' C++ bindings
+long_description    This C++ binding for 0mq/zmq is a 'high-level' library \
+                    that hides most of the c-style interface core 0mq provides. \
+                    It consists of a number of header and source files all residing in the zmq directory, \
+                    these files are provided under the MPLv2 license (see LICENSE for details).
+
+depends_lib         port:zmq
```

Comment:
When using portgroups, it is best practice to `-append` dependencies to not overwrite any portgroup dependencies.

Reply:
Done